examples/todomvc/app.re

```diff
@@ -6,7 +8,7 @@ let enterKey = 13;
 
 let namespace = "reason-react-todos";
 
-let saveLocally todos => ReasonJs.LocalStorage.setItem namespace (ReasonJs.JSON.stringify todos);
+let saveLocally todos => ReasonJs.LocalStorage.setItem namespace (ReasonJs.Json.stringify todos);
```

Comment:
Right, but since the module name is the file name with the first letter capitalized, then if using files as standalone modules you would have to name it "jSON.re". So that's where the current convention comes from, but that argument is kind of moot now with aliases.

I'd be happy to change it.